drivers/hubitat-mqtt-link-driver.groovy

```diff
@@ -274,15 +271,22 @@ def publishMqtt(topic, payload, qos = 0, retained = false) {
 // ========================================================
 
 def connected() {
-    info("[connected] Connected to broker")
-    sendEvent (name: "connectionState", value: "connected")
+    state?.reconnectDelay = 1
+    debug("[connected] Connected to broker")
+    sendEvent (name: "status", value: "connected")
     announceLwtStatus("online")
```

Comment:
The LWT appears to be broken now.  LWT always reports 'offline' even when connected to the broker and msgs are received.  Can you try to fix it so that LWT is reported accurately? 

![image](https://user-images.githubusercontent.com/2975433/103241408-1fba8d00-4921-11eb-9855-c8867399097c.png)

Reply:
Yeah, I agree LWT doesn't work anymore.  It seems a delay is needed of some sort after connection.  I've mostly given up on hubitat atm and am moving to home-assistant.  My hub was crashing in under 3 hours at one point last week.  I've disabled most all user apps/code to keep it running...